maps/journey/main.lua

```diff
@@ -11,6 +11,7 @@ local Functions = require 'maps.journey.functions'
 local Unique_modifiers = require 'maps.journey.unique_modifiers'
 local Map = require 'modules.map_info'
 local Global = require 'utils.global'
+local ClearVacantPlayers = require 'modules.clear_vacant_players'
```

Comment:
Hmm, my idea was not to force-load here, but conditional-load. I guess I forgot about that.

If this is changed to conditional loading:
```suggestion
local ClearVacantPlayers = is_loaded('modules.clear_vacant_players')
```
Then we could, uhm, offload (?) the decision of including the module to some other location. (I don't really know where...)

All the code for module `clear_vacant_players` in Journey is ready for the module to be null, with `if journey.clear_vacant_players then ... end`.
The whole Journey should work seamlessly.

Would this make the Journey changes more palatable?
I think I will extract the Journey changes to a different PR in any case. (Tomorrow?)
Separating them will allow for slower and more thoughtful decisions with-regards-to Journey.